woocommerce/woocommerce-ios
# [Login- "Continue with WordPress.com" flow] - Add `What is WordPress.com?` button.

Closes: #7212

- [ ] The changes done in [Authenticator library](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS) can be reviewed at https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/646

### Description
This PR adds `What is WordPress.com?` button in the Email entry screen of "Continue with WordPress.com" flow. 

As we cannot expect all users who install the app to be familiar with "WordPress.com" we are adding this button to teach users about "WordPress.com".  

This is part of "**Experiment 2: Add more information about the login process**" in pe5sF9-6C-p2

### Testing instructions 
- Install and launch the app.
- Tap "Continue With WordPress.com" button in the login prolougue screen.
- Ensure that you see "What is WordPress.com?" link under the "Continue" button.
- Tap on the "What is WordPress.com?" link and validate that it opens "https://woocommerce.com/document/what-is-a-wordpress-com-account/" webpage.

### Screenshots


| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/524475/177331100-1649a299-655b-47dc-ba14-f8ac5547dd0b.jpeg" width="400" /> | <img src="https://user-images.githubusercontent.com/524475/177331050-d53749ab-63dc-4009-8fe6-17541a8ff705.jpeg" width="400" />  |

---
